docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+---
+version: "3.9"
```

Comment:
My Debian 11.5 complains about the version being too high. 

```
$ docker-compose -v
docker-compose version 1.25.0, build unknown
$ docker --version
Docker version 20.10.5+dfsg1, build 55c4c88
```

Using version: "3" does the trick for me. Works like a charm. And it also has to be 'docker-compose' in the docs.

I assume you are using a newer version of Docker. Does it work for you, if you switch these bits, so that older systems can use this as well?